updates/location_unsupported_assets/v1.json

```diff
@@ -0,0 +1,11 @@
+{
+  "location_unsupported_assets": {
+    "insert": {
+      "binance": [
+        "BOME",
+        "SAGA",
+        "TNSR"
```

Comment:
this one isn't https://github.com/rotki/assets/blob/f00319b59898d0f63b0d475d6dbf41e2f582e3db/updates/24/updates.sql#L139